homework.md

```diff
@@ -86,28 +87,28 @@ You send the changes of your local repository to Github
 
 What is Github?
 
-A place for us to store and share our code.
+is a source control platform use when created code .is a store place where you can save your project
 
```

Comment:
I am not sure whether you need Github to be able to create code. Anyway, You should look for a better definition. 
I can suggest this definition: " it is a Git repository hosting service, that also provides access control and several collaboration features". But, there is always a better one so, please do your own research.